wifiphisher/common/recon.py

```diff
@@ -1,116 +1,60 @@
-"""
-This module handles all the operations regarding locating all the
-available access points
-"""
-
-from __future__ import division
-import threading
-import time
-import logging
+"""his module handles all access point the operations."""
+
+from __future__ import (absolute_import, division, print_function)
+from threading import Thread
+from time import (strftime, sleep)
+from logging import getLogger
 import scapy.layers.dot11 as dot11
-import wifiphisher.common.constants as constants
+from wifiphisher.common.constants import (NON_CLIENT_ADDRESSES,
+                                          ALL_2G_CHANNELS, LOCS_DIR)
```

Comment:
I remember agreeing a year (or two) back that we should import constants as constants. Something changed?

Reply:
No before we had `from wifiphisher.common.constants import *` which would bring everything and hence was confusing. With this however you can always search for them easily.

Reply:
Okay, agreed.